src/components/Proposals/DraftProposal/DraftProposalPage.tsx

```diff
@@ -206,6 +246,10 @@ const DraftProposalsPageContent = memo(
                 proposalType === ProposalType.Tactical
                   ? quorumThreshold
                   : undefined,
+              approvalThreshold:
+                proposalType === ProposalType.Tactical
```

Comment:
Likewise here, should this be

```suggestion
                proposalType !== ProposalType.Standard
```